### ISSUE TITLE
CustomFeature - A new feature to support ECDIS Chart 1

### ISSUE THREAD

**Issue by alvarosanuy** [S-101-Portrayal-subWG/Working-Documents#169]
**Refer to S101PT13 Paper S-101PT13-07.2**

**First Draft:**
[S-101PT13-07.2 New Feature Custom Feature FINAL 20240509.docx](https://github.com/S-101-Portrayal-subWG/Working-Documents/files/15268933/S-101PT13-07.2.New.Feature.Custom.Feature.FINAL.20240509.docx)

- [ ] **NIWC** to commence working on associated PC changes to support testing at the earliest opportunity (once accompanying draft FC becomes available)

**Comment by DavidGrant-NIWC** [comment]
Current status:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/8eb6af3c-bc45-4ba4-8cc6-590b3178a6a5)

**Comment by alvarosanuy** [comment]
**Input from PsWG meeting on 10/7/2024**

- [x] @DavidGrant-NIWC to talk to @kusala9 & @JeffWootton regarding amending the modelling of **Chart1Feature** to include an attribute (an existing, registered, one is the preference) to be used during Pick Report to describe, in plain english, the symbol represented by **drawingInstruction**.
- [ ] @kusala9 - Inform @DavidGrant-NIWC when the ECDIS Chart 1 TDS is finalised and ready for final performance of the PC changes in onShore ECDIS.
- [ ] @DavidGrant-NIWC - to update this GitHub issue when the task can be closed.

**Comment by alvarosanuy** [comment]
> [x] @DavidGrant-NIWC to talk to @kusala9 & @JeffWootton regarding amending the modelling of **Chart1Feature** to include an attribute (an existing, registered, one is the preference) to be used during Pick Report to describe, in plain english, the symbol represented by **drawingInstruction**.

**Not required** as the feature already has **featurename** in its modelling.

**Comment by kusala9** [comment]
agree. Dave and I have agreed this. There's a version of the cell above which has the text labels (done using featureName) and the pick report can use nameUsage != 1 to represent the textual information. We should make drawingInstruction private (but that may have to wait until 1.5.0?). Both the textual description and private visibility are enhancements on what is provided in S-57 though.

![image](https://github.com/user-attachments/assets/9c78da76-a1e9-455d-84ce-e3be2fd3b773)

**Comment by alvarosanuy** [comment]
**Close - Implemented in PC 1.4.0**
